### PR TITLE
Get the Webserver Working Again, After All the Tool Updates

### PR DESCRIPTION
**Pull request by natestuder**
These commits fix a bunch of issues in the webserver build in order to get the demo up and running again.

### CODE REVIEW

File: FreeRTOS-Plus/Demo/FreeRTOS_Plus_FAT_TCP_CGI_RISC-V/Makefile [hunk 1]
```diff
@@ -246,7 +246,7 @@ $(SDK_BUILD_DIR)/%.o : %.S
 $(OUT_ELF): $(OUT_OBJS) $(LINKER_SCRIPT) $(LINK_DEPS) libxil
 	@echo 'Building target: $@'
 	@echo '_______________________________'
-	$(GCC) $(CFLAGS) $(INCLUDES) $(OUT_OBJS) -o $@ $(LDFLAGS) $(LIBXIL_LDFLAGS) $(LDOPTS) -L$(LIBXIL_BUILD_DIR)
+	$(GCC) -v $(CFLAGS) $(INCLUDES) $(OUT_OBJS) -o $@ $(LDFLAGS) $(LIBXIL_LDFLAGS) $(LDOPTS) -L$(LIBXIL_BUILD_DIR)
```
Comment (arunthomas) [inline FreeRTOS-Plus/Demo/FreeRTOS_Plus_FAT_TCP_CGI_RISC-V/Makefile:249]: Do we need the -v?

File: FreeRTOS/Source/portable/GCC/RISC-V/portexception.h [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #define portRV_LOAD_ACCESS_FAULT 0x5
 #define portRV_STORE_ADDRESS_MISALIGNED 0x6
 #define portRV_STORE_ACCESS_FAULT 0x7
+#define portRV_POLICY_VIOLATION_FAULT 0x11
```
Comment (arunthomas) [inline FreeRTOS/Source/portable/GCC/RISC-V/portexception.h:11]: Change to custom use exception code.

File: FreeRTOS-Plus/Demo/FreeRTOS_Plus_FAT_TCP_CGI_RISC-V/Makefile (hunk 1 above)
Comment (natestuder) [reply FreeRTOS-Plus/Demo/FreeRTOS_Plus_FAT_TCP_CGI_RISC-V/Makefile:249]: No.  Debuging change left in.  Will remove.

File: FreeRTOS/Source/portable/GCC/RISC-V/portexception.h (hunk 2 above)
Comment (natestuder) [reply FreeRTOS/Source/portable/GCC/RISC-V/portexception.h]: Handled in other PR.

**Pull request merged**